Classes/PlayerEvent.swift

```diff
@@ -165,7 +165,7 @@ import Foundation
     
     // `AdCuePointsUpdate` event is received when ad cue points were updated. only sent when there is more then 0.
     class AdCuePointsUpdate: AdEvent {
-        convenience init(adCuePoints: AdCuePoints) {
+        convenience init(adCuePoints: PKAdCuePoints) {
```

Comment:
@gal-orlanczyk  let's make sure Android team have same prefix for this class.

cc @OrenMe